### DISCUSSION TITLE
Confused about ~shareCss for setting theme for shared notes

### DISCUSSION THREAD

**Discussion by fliptoback** [TriliumNext/Trilium#4368] [Q&A] [answered]
Hi, the Trilium wiki has a guide on styling the shared notes.
https://github.com/zadam/trilium/wiki/Sharing

I tried to follow it but nothing works so I think I am missing some important steps.

These are the steps I have so far:

1. Share out a particular notebook.
2. Create a subpage in that notebook. I named that as share.css
3. In that share.css, I set the Note Type to CSS, and then i pasted the CSS scripts from the theme.

Then I am stuck.

What do i do with this ~shareCss tag and what do i do with the inheritable label?

Please help. 

Thanks and much appreciated.

**Comment by zerebos** [comment]
You should set the `~shareCss` relation (pointing to that css note) on the top level of the notebook and make it inheritable.

1. Add new relation
![image](https://github.com/zadam/trilium/assets/6865942/d2c06454-dec4-4f49-9a7b-ec92fde65e4d)

2. Set the related note
![image](https://github.com/zadam/trilium/assets/6865942/2c11c85e-e955-4f76-86f9-bd3c9d76027c)

**Reply by fliptoback** [reply]
Hi Rauenzi,

Thanks for your help.

I followed your steps - but it is not working. Here are some screenshots.

I am trying to share out the folder called Wiki Test

This is how I have defined the share.css css scripts.
![image](https://github.com/zadam/trilium/assets/37237951/fc96555f-5913-479e-ac09-033d7ff133e4)

And on the main Wiki Test folder, I have set this as per your instructions:
![image](https://github.com/zadam/trilium/assets/37237951/37d9ff33-6cbe-48b5-bccf-e0943611a99d)

![image](https://github.com/zadam/trilium/assets/37237951/e96b5a96-3c3a-4b0c-bcbe-ef0379262fa5)


When I load the share note via its URL, I see this:
![image](https://github.com/zadam/trilium/assets/37237951/5edf9a31-1b3c-440c-ac3b-0f12e534cd52)

Any ideas?

**Reply by zerebos** [reply, marked as answer]
The theme you've put in the css note is a Trilium theme, not a share css theme. You might try one of these:

 - https://github.com/FrostMiKu/Share.CSS
 - https://github.com/zadam/trilium/discussions/2681

**Reply by fliptoback** [reply]
Thanks Rauenzi.

Changing that theme to the paper css now works.

I didnt know that the CSS theme for shared notes and for the overall trilium app is actually different. Thanks for pointing that out - and also letting me know how to apply the shareCSS tag.

Thanks again.
